build.gradle

```diff
@@ -80,6 +80,11 @@ jar {
     from {
     	configurations.extraLibs.collect { it.isDirectory() ? it : zipTree(it) }
     }
+    manifest {
```

Comment:
Is this `manifest`block of code necessary to the library of was it only necessary for local tests? If the latter is true, it must be removed from the final version.

Reply:
It is not just for testing, but a general convenience. According to https://docs.oracle.com/javase/tutorial/deployment/jar/downman.html  "By using the Class-Path header in the manifest, you can avoid having to specify a long -classpath flag when invoking Java to run the your application."

Reply:
CST is not an application, it is a library. Differently from TROCA, it will never be run as a Java JAR application, but only be imported into applications from the maven repository. This is not needed.

Reply:
I really included that with the hope to make dependency transitivity to work with gradle, while these libraries (CST and MECA) are local files and not maven deployed. But in fact this didn't worked for that purpose. I believe this could be removed without too much pain. Nevertheless, I once got in trouble due to a classpath which was longer than the 255 characters limit in a Windows machine, and this was precluding the use of the library in question (this was not CST, by the way). If CST is meant to be used with other libraries, this MIGHT be a problem, as the CST classpath is already big due to its many dependencies. Creating this entry in the manifest might be a possible solution to the classpath size and this 255 characters limitation in Windows machines. This is the reason I left it there.